src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -183,22 +183,12 @@ html .breakpoints-list .breakpoint.paused {
   inset-inline-end: 15px;
   inset-inline-start: auto;
   position: absolute;
-  top: 8px;
-  display: none;
+  top: -100px;
```

Comment:
We should probably add a comment to explain this value. Along the lines of "hide the button outside of the row until hovered or focused", maybe.